src/config/database.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose");
+
+const connectDB = async () => {
+  await mongoose.connect(
+    // here we are connecting to the cluster if we dont name anything after '/' || if we add the database name after the '/' then will connect to specific database
+    "mongodb+srv://Sanskriti:21sanskriti@namastenode.hodnj.mongodb.net/devConnect"
```

Comment:
why this is available for anyone to use on the github, anyone can connect to this DB and screw up things

use env vars for storing such secret info